source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz/custom_solution_for_self_managed_user_system/quick_start_guide_service_apps.rst

```diff
@@ -1,6 +1,34 @@
 .. _quick_start_guide_service_apps:
 
-Quick Start Guide - Service Apps
-================================
+Quick Start Guide
```

Comment:
Revert the title to what I had. I believe it makes the context of this clear, because there is no "Service Apps" menu option that comes before this. For the other quick start guides, the prior menu option provides the necessary context and therefore I kept them as just "Quick Start Guide'.